src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -1,6 +1,6 @@
 ////////////////////////////////////////////////////////////////////////////////
 // checkstyle: Checks Java source code for adherence to a set of rules.
-// Copyright (C) 2001-2014  Oliver Burn
+// Copyright (C) 2001-2020 the original author or authors.
```

Comment:
Can you update command at https://github.com/checkstyle/checkstyle/wiki/How-to-make-a-release#before-starting to find and "*.g" files also

Reply:
To confirm, is this correct:
    ```
    find . -type f \( -name *.java -o -name *.header -o -name *.g \) -exec sed -i  \
      "s/\/\/ Copyright (C) 2001-2015 the original author or authors./\/\/ Copyright (C) 2001-2016 the original author or authors./g" \
      {} +
    ```

Reply:
yes, correct.
I updated wiki.

Reply:
I think it make sense to move it `.ci` folder as script to ease operation by multiple maintainers.
current and previous year is easy to get by shell commands.